plugins/discovery-gce/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -0,0 +1,2 @@
+ALL-UNNAMED:
+  - set_https_connection_properties # required by google-http-client
```

Comment:
We should port over [the comments from the SM policy file](https://github.com/elastic/elasticsearch/blob/e454d74de4e7eaf7054c3992632b4004e864ca7c/modules/repository-gcs/src/main/plugin-metadata/plugin-security.policy#L15).